lambda-runtime/src/AWS/Lambda/Runtime.hs

```diff
@@ -0,0 +1,40 @@
+module AWS.Lambda.Runtime
+  ( run
+  )
+where
+
+import AWS.Prelude
+import Control.Monad.Except (runExceptT)
+
+import qualified AWS.Lambda.Client as Client
+import qualified Data.Aeson        as JSON
+
+run
+  :: forall env m . (MonadCatch m, MonadIO m)
+  => env
+  -> (JSON.Value -> RIO env JSON.Value)
+  -> m ()
+run env lambdaFn = forever . runtimeLoop $ (runRIO env . lambdaFn)
+
+runtimeLoop
```

Comment:
This is misnamed, it does *one* request / response cycle. Its not the loop. the loop is in `run`.

I recommend: `processEvent`.

Reply:
Yes, I agree. Going to make this change.